reference-architecture/gce-ansible/playbooks/openshift-setup.yaml

```diff
@@ -35,6 +35,7 @@
     openshift_master_cluster_hostname: "internal-openshift-master.{{ public_hosted_zone }}"
     openshift_master_cluster_public_hostname: "openshift-master.{{ public_hosted_zone }}"
     osm_default_subdomain: "{{ wildcard_zone }}"
+    openshift_master_default_subdomain: "{{osm_default_subdomain}}"
```

Comment:
I think this will be find, since I suspect that osm_default_subdomain is being explicitly prior to this playbook being called.